### PR TITLE
[Web Share] Add list of usually shareable file types

### PR DESCRIPTION
**Pull request by tomayac**
#### Summary
Add note about Chromium's list of accepted file types for `navigator.share()`.

#### Motivation
 Questions about this keep coming up.

#### Supporting details
https://docs.google.com/document/d/1tKPkHA5nnJtmh2TgqWmGSREUzXgMUFDL6yMdVZHqUsg/edit

#### Metadata

- [ ] Adds a new document
- [X] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Elchi3** [comment]
This seems very useful but linking to a google doc makes me cringe a bit. Should we include a table with this information in the docs directly (in which we invite other browsers to talk about their accepted file types, too)?

**Comment by tomayac** [comment]
I like this idea! Here's the Markdown for such a table. Please advise where this would go.


```md
| Category    | Extension - `MIME type                           |
| ----------- | ------------------------------------------------ |
| Application |                                                  |
|             | `.pdf` - `application/pdf` (from Chromium 93)    |
| Audio       |
|             | `.flac` - `audio/flac`                           |
|             | `.m4a` - `audio/x-m4a`                           |
|             | `.mp3` - `audio/mpeg` (also accepts `audio/mp3`) |
|             | `.oga` - `audio/ogg`                             |
|             | `.ogg` - `audio/ogg`                             |
|             | `.opus` - `audio/ogg`                            |
|             | `.wav` - `audio/wav`                             |
|             | `.weba` - `audio/webm`                           |
| Image       |                                                  |
|             | `.bmp` - `image/bmp`                             |
|             | `.gif` - `image/gif`                             |
|             | `.ico` - `image/x-icon`                          |
|             | `.jfif` - `image/jpeg`                           |
|             | `.jpeg` - `image/jpeg`                           |
|             | `.jpg` - `image/jpeg`                            |
|             | `.pjp` - `image/jpeg`                            |
|             | `.pjpeg` - `image/jpeg`                          |
|             | `.png` - `image/png`                             |
|             | `.svg` - `image/svg+xml`                         |
|             | `.svgz` - `image/svg+xml`                        |
|             | `.tif` - `image/tiff`                            |
|             | `.tiff` - `image/tiff`                           |
|             | `.webp` - `image/webp`                           |
|             | `.xbm` - `image/x-xbitmap`                       |
| Text        |                                                  |
|             | `.css` - `text/css`                              |
|             | `.csv` - `text/csv`                              |
|             | `.ehtml` - `text/html`                           |
|             | `.htm` - `text/html`                             |
|             | `.html` - `text/html`                            |
|             | `.shtm` - `text/html`                            |
|             | `.shtml` - `text/html`                           |
|             | `.text` - `text/plain`                           |
|             | `.txt` - `text/plain`                            |
| Video       |                                                  |
|             | `.m4v` - `video/mp4`                             |
|             | `.mp4` - `video/mp4`                             |
|             | `.mpeg` - `video/mpeg`                           |
|             | `.mpg` - `video/mpeg`                            |
|             | `.ogm` - `video/ogg`                             |
|             | `.ogv` - `video/ogg`                             |
|             | `.webm` - `video/webm`                           |

```

**Comment by Elchi3** [comment]
It seems important enough that I'd create a new h2 before the "Examples" section like so:

> ## Shareable file types 
> 
> table
> 
> ## Examples
> ...

And instead of your note with the google doc link, something like this:

> - `files`: An array of {{domxref("File")}} objects representing files to be shared. See below for shareable file types.

File: files/en-us/web/api/navigator/share/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ navigator.share(data)
     - `url`: A {{domxref("USVString")}} representing a URL to be shared.
     - `text`: A {{domxref("USVString")}} representing text to be shared.
     - `title`: A {{domxref("USVString")}} representing a title to be shared. May be ignored by the target.
-    - `files`: An array of {{domxref("File")}} objects representing files to be shared.
+    - `files`: An array of {{domxref("File")}} objects representing files to be shared. See [below](#shareable-file-types) for shareable file types.
```
Comment (tomayac) [inline files/en-us/web/api/navigator/share/index.md:41]:
```suggestion
    - `files`: An array of {{domxref("File")}} objects representing files to be shared. See [below](#shareable_file_types) for shareable file types.
```

File: files/en-us/web/api/navigator/share/index.md [hunk 2]
```diff
@@ -64,6 +64,56 @@ The {{jsxref("Promise")}} may be rejected with one of the following `DOMExceptio
 - `DataError`
   - : There was a problem starting the share target or transmitting the data.
 
+## Shareable file types
+
+| Category    | Extension - `MIME type                           |
```
Comment (tomayac) [inline files/en-us/web/api/navigator/share/index.md:69]:
```suggestion
| Category    | Extension - MIME type                            |
```

File: files/en-us/web/api/navigator/share/index.md [hunk 3]
```diff
@@ -64,6 +64,56 @@ The {{jsxref("Promise")}} may be rejected with one of the following `DOMExceptio
...
+|             | `.txt` - `text/plain`                            |
+| Video       |                                                  |
+|             | `.m4v` - `video/mp4`                             |
+|             | `.mp4` - `video/mp4`                             |
+|             | `.mpeg` - `video/mpeg`                           |
+|             | `.mpg` - `video/mpeg`                            |
+|             | `.ogm` - `video/ogg`                             |
+|             | `.ogv` - `video/ogg`                             |
+|             | `.webm` - `video/webm`                           |
+
```
Comment (Elchi3) [inline files/en-us/web/api/navigator/share/index.md:116]: Seeing the table rendered now makes it look quite bulky to me. Maybe a list would be better, what do you think?
 Also, an introductory sentence what this list is would be helpful.


```suggestion

The following is a list of usually shareable file types. However, you should always test with {{domxref("navigator.canShare()")}} if sharing would succeed.

- Application
  - `.pdf` - `application/pdf` (from Chromium 93)
- Audio
  - `.flac` - `audio/flac`
  - `.m4a` - `audio/x-m4a`
  - `.mp3` - `audio/mpeg` (also accepts `audio/mp3`)
  - `.oga` - `audio/ogg`
  - `.ogg` - `audio/ogg`
  - `.opus` - `audio/ogg`
  - `.wav` - `audio/wav`
  - `.weba` - `audio/webm`
- Image
  - `.bmp` - `image/bmp`
  - `.gif` - `image/gif`
  - `.ico` - `image/x-icon`
  - `.jfif` - `image/jpeg`
  - `.jpeg` - `image/jpeg`
  - `.jpg` - `image/jpeg`
  - `.pjp` - `image/jpeg`
  - `.pjpeg` - `image/jpeg`
  - `.png` - `image/png`
  - `.svg` - `image/svg+xml`
  - `.svgz` - `image/svg+xml`
  - `.tif` - `image/tiff`
  - `.tiff` - `image/tiff`
  - `.webp` - `image/webp`
  - `.xbm` - `image/x-xbitmap`
- Text
  - `.css` - `text/css`
  - `.csv` - `text/csv`
  - `.ehtml` - `text/html`
  - `.htm` - `text/html`
  - `.html` - `text/html`
  - `.shtm` - `text/html`
  - `.shtml` - `text/html`
  - `.text` - `text/plain`
  - `.txt` - `text/plain`
- Video
  - `.m4v` - `video/mp4`
  - `.mp4` - `video/mp4`
  - `.mpeg` - `video/mpeg`
  - `.mpg` - `video/mpeg`
  - `.ogm` - `video/ogg`
  - `.ogv` - `video/ogg`
  - `.webm` - `video/webm`
```

File: files/en-us/web/api/navigator/share/index.md [hunk 4]
```diff
@@ -64,6 +64,55 @@ The {{jsxref("Promise")}} may be rejected with one of the following `DOMExceptio
...
+  - `.text` - `text/plain`
+  - `.txt` - `text/plain`
+- Video
+  - `.m4v` - `video/mp4`
+  - `.mp4` - `video/mp4`
+  - `.mpeg` - `video/mpeg`
+  - `.mpg` - `video/mpeg`
+  - `.ogm` - `video/ogg`
+  - `.ogv` - `video/ogg`
+  - `.webm` - `video/webm`
```
Comment (tomayac) [inline files/en-us/web/api/navigator/share/index.md:115]:
```suggestion
  - `.webm` - `video/webm`

```

**Review by Elchi3 — APPROVED** [review]
Danke! 👍

**Comment by tomayac** [comment]
Woohoo, thanks for the review and making this even better! 

Viele Grüße aus Hamburg!

File: files/en-us/web/api/navigator/share/index.md [hunk 5]
```diff
@@ -64,6 +64,56 @@ The {{jsxref("Promise")}} may be rejected with one of the following `DOMExceptio
 - `DataError`
   - : There was a problem starting the share target or transmitting the data.
 
+## Shareable file types
+
+The following is a list of usually shareable file types. However, you should always test with {{domxref("navigator.canShare()")}} if sharing would succeed.
+
+- Application
+  - `.pdf` - `application/pdf` (from Chromium 93)
```
Comment (hamishwillee) [inline files/en-us/web/api/navigator/share/index.md:72]: @Elchi3 FWIW Little concerned that the list might end up being "Chromium" specific, and yet one more list of things that has to be kept updated outside of BCD. Maybe remove the "(from Chromium 93)."

I wonder if this list is just "all the media types you can play/use on the platform". I wonder what the things you _can't send_ are? If so, we might be better off saying "Generally most media and text files that can be played on the platform can also be shared".

Comment (tomayac) [inline files/en-us/web/api/navigator/share/index.md:72]:
```suggestion
  - `.pdf` - `application/pdf`
```

**Pull request merged**